tools/olm-bundle/Makefile

```diff
@@ -0,0 +1,11 @@
+PROJECT_NAME=olm-bundle
+
+BUNDLE_VERSION ?= $(shell cat $(dirname $(realpath $0))../../release.version | cut -d'-' -f1)
+.DEFAULT_GOAL := default
+
+include ../../Makefile.docker
+
+olm_install:
+	./generate-olm-bundle.sh BUNDLE_VERSION=$(BUNDLE_VERSION) DOCKER_REGISTRY=$(DOCKER_REGISTRY) DOCKER_ORG=$(DOCKER_ORG) DOCKER_TAG=$(DOCKER_TAG)
+
+default: olm_install
```

Comment:
The intented benefit of running the `make` command is to use the Docker env vars defined in the `../../Makefile.docker` file: `DOCKER_REGISTRY`, `DOCKER_ORG`, and `DOCKER_TAG` so users can create bundles based on the container images they have just built.

I have just plugged it into the main Makefile now so that users can run:
```
make olm_install
```
to generate their bundle files